File: src/main/java/ru/arc/ai/assistant/ToolMessageRaw.java
package ru.arc.ai.assistant;

import com.google.gson.JsonElement;
import lombok.Builder;
import lombok.Data;

import java.util.UUID;

@Data
@Builder
public class ToolMessageRaw {
    public UUID uuid;
    public String toolName;
    public JsonElement toolDto;
    public long timestamp;
}

File: src/main/kotlin/ru/arc/ai/assistant/ToolMessageRaw.kt
package ru.arc.ai.assistant

import com.google.gson.JsonElement
import java.util.UUID

data class ToolMessageRaw(
    val uuid: UUID? = null,
    val toolName: String? = null,
    val toolDto: JsonElement? = null,
    val timestamp: Long = 0L
)
